mobile/src/bundles/chat/screens/chat-list/chat-list.tsx

```diff
@@ -78,10 +86,32 @@ const ChatList: React.FC = () => {
         [navigation],
     );
 
+    const handleLogout = (): void => {
+        void dispatch(logout());
+    };
+
     return (
         <View style={globalStyles.flex1}>
-            <View style={[globalStyles.p25, styles.header]}>
+            <View
+                style={[
+                    globalStyles.p25,
+                    globalStyles.pr15,
+                    globalStyles.flexDirectionRow,
+                    globalStyles.justifyContentSpaceBetween,
+                    globalStyles.alignItemsCenter,
+                    styles.header,
+                ]}
+            >
                 <Text category={TextCategory.H3}>Chat</Text>
+                <Button
```

Comment:
We will have a message about approval on profile pages, where the logout button is. Logout button looks in the right place, but how to combine it with the message?

Reply:
Then it's probably better to remove Logout from this page.
From here, you can go to the bottom of the navigation to Chat or Candidates and then log out.